webhook_to_fedora_messaging/endpoints/models/service.py

```diff
@@ -0,0 +1,57 @@
+from abc import ABC
+from datetime import datetime
+from typing import List, Optional
+
+from pydantic import BaseModel
+
+
+class ServiceBase(BaseModel, ABC):
+    """
+    Base: Service
+    """
+
+    class Config:
+        from_attributes = True
+
+
+class ServiceExternal(ServiceBase):
+    uuid: str
+    name: str
+    type: str
+    desc: str
+    token: str
+    user_id: int
+    creation_date: datetime
+
+
+class ServiceInternal(ServiceExternal):
+    id: str
+
+
+class ServiceRequestMain(BaseModel):
+    name: str
+    type: str
```

Comment:
This could actually be an enumeration of the possible types: right now only `github`. This way, there's no risk someone creates a service for a backend we don't support (or makes a typo)